### PR TITLE
vasu.is-a.dev domain registrar

### PR DESCRIPTION
**Pull request by Vasu-xD**
domain registration request...

### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/vasu.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/", "https://vasuxd.github.io/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD", "VasuXD",
+    "email": "mr.v4su@gmail.com"
+    "telegram": "VasuXD","Vasu_xD",
+    "github": "VasuXD","Vasu-xD",
+  },
+  "record": {
+    "CNAME": "vasu-xd.github.io", "vasuxd.github.io",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:11]:
```suggestion
    "CNAME": "vasu-xd.github.io"
```

File: domains/vasu.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/", "https://vasuxd.github.io/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD", "VasuXD",
+    "email": "mr.v4su@gmail.com"
+    "telegram": "VasuXD","Vasu_xD",
+    "github": "VasuXD","Vasu-xD",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:8]:
```suggestion
    "github": "VasuXD"
```

File: domains/vasu.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/", "https://vasuxd.github.io/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD", "VasuXD",
+    "email": "mr.v4su@gmail.com"
+    "telegram": "VasuXD","Vasu_xD",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:7]:
```suggestion
    "telegram": "VasuXD",
```

File: domains/vasu.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/", "https://vasuxd.github.io/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD", "VasuXD",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:5]:
```suggestion
    "username": "Vasu-xD",
```

File: domains/vasu.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/", "https://vasuxd.github.io/VasuXD/",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:3]:
```suggestion
  "repo": "https://github.com/vasu-xd/VasuXD/",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/vasu.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  "description": "Vasu's personal developer website",
+  "repo": "https://github.com/vasu-xd/VasuXD/",
+  "owner": {
+    "username": "Vasu-xD",
+    "email": "mr.v4su@gmail.com"
+    "telegram": "VasuXD",
+    "github": "VasuXD"
+  },
+  "record": {
+    "CNAME": "vasu-xd.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:11]: I can't seem to reach this site.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/vasu.json (hunk 6 above)
Comment (Vasu-xD) [reply domains/vasu.json:11]: now okay! 

👉🏻 https://vasu-xd.github.io/Vasu-xD/

File: domains/vasu.json [hunk 7]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Vasu's personal developer website",
+  "repo": "https://vasu-xd.github.io/Vasu-xD/",
```
Comment (WilliamDavidHarrison) [inline domains/vasu.json:3]:
```suggestion
  "repo": "https://github.com/vasu-xd/Vasu-xD/",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/vasu.json (hunk 7 above)
Comment (Vasu-xD) [reply domains/vasu.json]: ok done!

**Review by wdhdev — DISMISSED** [review]

**Comment by Vasu-xD** [comment]
when pull request will be merged?

**Comment by WilliamDavidHarrison** [comment]
> when pull request will be merged?

Most likely in a few days.

I'm only a helper so I cannot merge PR's unfortunately.

**Comment by Vasu-xD** [comment]
How much time have to wait? 🙄
@WilliamDavidHarrison 
@is-a-dev-bot 
@is-a-dev

**Comment by WilliamDavidHarrison** [comment]
> How much time have to wait? 🙄
> @WilliamDavidHarrison
> @is-a-dev-bot
> @is-a-dev

Please don't ping us, it'll only take longer

**Review by MaskDuck — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**